balancer-js/src/modules/pools/pool-types/concerns/composableStable/join.concern.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { expect } from 'chai';
```

Comment:
Remove this. Eventually we can replace with an integration test similar to: https://github.com/balancer-labs/balancer-sdk/blob/develop/balancer-js/src/modules/pools/pool-types/concerns/weighted/join.concern.integration.spec.ts